engine/table/src/main/java/io/deephaven/engine/table/impl/by/MinMaxHelper.java

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.engine.table.impl.by;
+
+import io.deephaven.util.QueryConstants;
+
+final class MinMaxHelper {
```

Comment:
Same as the other helpers.

Reply:
I can justify removing `CompactKernelImpl` and reverting to the previous code for the CompactKernel replication b/c it already exists, but I can't justify writing new non-trivial replication code. I'm happy to move `isNullOrNan` into some common place like `DoubleComparisons` (or somewhere else accessible) if you prefer.

Reply:
I think the example provided by the compaction kernels shows that this case is trivial, unless I'm mistaken.